src/styles/index.scss

```diff
@@ -1,3 +1,39 @@
-body {
+@use './blocks/header.scss';
+@use './blocks/card.scss';
+@use './blocks/stars.scss';
+@use './utils/variables' as *;
+
+@mixin column-count($count) {
+  grid-template-columns: repeat($count, $card-width);
+}
+
+* {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  box-sizing: border-box;
+  color: $black-color;
+}
+
+.main {
+  display: grid;
+  grid-template-columns: repeat(4, $card-width);
+  grid-template-rows: repeat(2);
```

Comment:
The `grid-template-rows` property is missing a value. Consider specifying the number of rows or the row height to ensure the grid layout functions as intended.